CMakeLists.txt

```diff
@@ -321,7 +321,7 @@ if(SOFA_INSTALL_RESOURCES_FILES)
 endif()
 
 file(WRITE "${CMAKE_BINARY_DIR}/plugins/README.txt" "This folder will be automatically scanned by the Plugin Manager.")
-install(DIRECTORY plugins/ DESTINATION plugins COMPONENT resources)
+install(DIRECTORY applications/plugins/ DESTINATION plugins COMPONENT resources)
```

Comment:
Actually the bug is real, as stated by @jnbrunet and @marques-bruno but the fix should be
`install(DIRECTORY ${CMAKE_BINARY_DIR}/plugins/ DESTINATION plugins COMPONENT resources)`